mlflow/store/tracking/file_store.py

```diff
@@ -231,7 +231,7 @@ def _get_param_path(self, experiment_id, run_uuid, param_name):
 
     def _get_experiment_tag_path(self, experiment_id, tag_name):
         _validate_experiment_id(experiment_id)
-        _validate_param_name(tag_name)
+        _validate_tag_name(tag_name)
```

Comment:
An experiment is not a run parameter, should be validated using `_validate_tag_name`.